src/parser/rogue/outlaw/modules/features/Checklist/Component.js

```diff
@@ -44,7 +44,7 @@ const OutlawRogueChecklist = ({ combatant, castEfficiency, thresholds }) => {
       </Rule>
       <Rule
         name="Use your finishers efficiently"
-        description={<>Your two damaging finishers should typically be used at maximum combo points. If you have <SpellLink id={SPELLS.RUTHLESS_PRECISION.id} /> active you should prioritize <SpellLink id={SPELLS.BETWEEN_THE_EYES.id} /> as your damaging finisher.</>}
+        description={<>Your two damage finishers should typically be used at maximum combo points. Whenever <SpellLink id={SPELLS.BETWEEN_THE_EYES.id} /> is off cooldown, you should always prioritize it over <SpellLink id={SPELLS.DISPATCH.id} />.</>}
       >
```

Comment:
This should definitely say "damaging" for grammar.

Reply:
Changed.